packages/terra-dialog-modal/package.json

```diff
@@ -0,0 +1,51 @@
+{
+  "name": "terra-dialog-modal",
+  "main": "lib/DialogModal.js",
+  "version": "0.0.0",
+  "description": "The Dialog Modal would allow for dynamic height modals. Dialog modal would be a limited use case, as dynamic heights breaks with more complicated DOM structures. If content is too complicated, the terra-modal-manager should be used. The components is placed at an 8000 z-index. The dialog supports release and request focus props similar to terra-popup and terra-date-picker, so it can be presented from another modal with focus.",
```

Comment:
Reminder to update this description with any changes made to description in readme